Butt Slaps

Sishen1542 opened the pull request:
A round of applause!  Same thing as face slapping, but now for your other cheeks. Target groin in disarm intent and click on some spaceman booty. RP has it, main needs it. Roleplay shall never be the same again.


![](http://puu.sh/DscmN/02beb5ad69.png)

edit:
Butt slaps 2.0
Added nympho to defined traits, added arousal damage to ass slapping (arousal system needs to be enabled), and added forced orgasms to nymphos at max arousal



![](http://puu.sh/Dt4aI/1616ddc36d.png)

Sishen1542 committed:
Booty Calls!

Added butt slaps

Sishen1542 committed:
Nympho Traiting & Arousal!

Butt slaps 2.0
Added nympho to defined traits, added arousal damage to ass slapping (arousal system needs to be enabled), and added forced orgasms to nymphos at max arousal

deathride58 commented on code/modules/mob/living/carbon/human/species.dm:
```diff
@@ -1463,6 +1463,19 @@ GLOBAL_LIST_EMPTY(roundstart_races)
+		user.visible_message("<span class='danger'>[user] slaps [target]'s ass!</span>",
+			"<span class='notice'>You slap [target]'s ass! </span>",\
+		"You hear a slap.")
+		stop_wagging_tail(target)
```
make this only stop wagging the tail if the target isn't a nymphomaniac

Sishen1542 replied:
Added, also added the nympho check for wagging to face slapping.

deathride58 commented on code/__DEFINES/traits.dm:
```diff
@@ -70,6 +70,7 @@
 #define	TRAIT_PHOTOGRAPHER		"photographer"
 #define	TRAIT_MUSICIAN			"musician"
 #define	TRAIT_CROCRIN_IMMUNE    "crocin_immune"
+#define TRAIT_NYMPHO			"nymphomania"
```
unused trait? you havent made anything in the code use this

Sishen1542 replied:
Couldn't figure out how to force orgasm with nymphos only without adding it in the define.
![](http://puu.sh/DtoA4/b45c7d22a6.png)

Sishen1542 committed:
Tail Wagging

Makes it so nymphos will continue wagging their tails when booty or face slapped
